PsfRuntime/CreateProcessHook.cpp

```diff
@@ -165,8 +175,12 @@ BOOL WINAPI CreateProcessFixup(
         startupInfo,
         processInformation))
     {
+        if (cnvtCmdLine != nullptr)

```

Comment:
Always use {} even if the block is 1 line.

However, if you use std::unique_ptr you don't have to wrry about manually deleting the array.